WordPress/openverse-catalog
# Make 'sound' category more specific

Related to wordpress/openverse-api#526

## Description


For audio, `sound` is too ambiguous a category name. This PR updates the provider scripts that were generating records with a `sound` category to use more specific categories instead:

* Freesound's `sound` has been updated to `sound_effect`
* Wikimedia's `sound` has been updated to `pronunciation`, since that's what was being checked for

**NOTE**: We will also need to go back and alter the existing catalog records with these values. This can be done with the following commands:

```sql
UPDATE audio
SET category = (
    CASE
        WHEN provider='freesound' AND category='sound' THEN NULL
        WHEN provider='wikimedia_audio' AND category='sound' THEN 'pronunciation'
    END
);
```

## Testing Instructions

1. `just test`
1. Optionally:
    1. Run the Freesound DAG until some media can be ingested.
    1. Ensure that there are no records with the category `sound` in the audio table.

## Checklist


[best_practices]:https://git-scm.com/book/en/v2/Distributed-Git-Contributing-to-a-Project#_commit_guidelines

## Developer Certificate of Origin


<details>
<summary>Developer Certificate of Origin</summary>

```
Developer Certificate of Origin
Version 1.1

Copyright (C) 2004, 2006 The Linux Foundation and its contributors.
1 Letterman Drive
Suite D4700
San Francisco, CA, 94129

Everyone is permitted to copy and distribute verbatim copies of this
license document, but changing it is not allowed.


Developer's Certificate of Origin 1.1

By making a contribution to this project, I certify that:

(a) The contribution was created in whole or in part by me and I
    have the right to submit it under the open source license
    indicated in the file; or

(b) The contribution is based upon previous work that, to the best
    of my knowledge, is covered under an appropriate open source
    license and I have the right under that license to submit that
    work with modifications, whether created in whole or in part
    by me, under the same open source license (unless I am
    permitted to submit under a different license), as indicated
    in the file; or

(c) The contribution was provided directly to me by some other
    person who certified (a), (b) or (c) and I have not modified
    it.

(d) I understand and agree that this project and the contribution
    are public and that a record of the contribution (including all
    personal information I submit with it, including my sign-off) is
    maintained indefinitely and may be redistributed consistent with
    this project or the open source license(s) involved.
```

</details>
